### DISCUSSION TITLE
failed deployment: RangeError: Maximum call stack size exceeded

### DISCUSSION THREAD

**Discussion by hewb-dev-team** [vercel/community#5762] [Help] [answered]
### Summary

I'm having this error while trying to deploy on vercel 

RangeError: Maximum call stack size exceeded
--
14:07:18.008 | at parse (/vercel/path0/node_modules/next/dist/compiled/micromatch/index.js:15:6313)
14:07:18.008 | at picomatch.makeRe (/vercel/path0/node_modules/next/dist/compiled/micromatch/index.js:15:21670)
14:07:18.008 | at picomatch (/vercel/path0/node_modules/next/dist/compiled/micromatch/index.js:15:19637)
14:07:18.008 | at /vercel/path0/node_modules/next/dist/compiled/micromatch/index.js:15:19294
14:07:18.008 | at Array.map (<anonymous>)
14:07:18.009 | at picomatch (/vercel/path0/node_modules/next/dist/compiled/micromatch/index.js:15:19286)
14:07:18.009 | at micromatch.isMatch (/vercel/path0/node_modules/next/dist/compiled/micromatch/index.js:15:1090)
14:07:18.009 | at /vercel/path0/node_modules/next/dist/build/collect-build-traces.js:245:48
14:07:18.009 | at shouldIgnore (/vercel/path0/node_modules/next/dist/build/collect-build-traces.js:75:9)
14:07:18.010 | at /vercel/path0/node_modules/next/dist/build/collect-build-traces.js:86:23
14:07:18.042 | Error: Command "next build" exited with 1

### Steps to Reproduce

<img width="1486" alt="Screenshot 2024-02-07 at 14 22 16" src="https://github.com/vercel/community/assets/156811434/a24faa3d-d5a3-4a85-a66d-9db7cca3b8eb">

deploy to vercel

**Comment by hewb-dev-team** [comment]
<img width="1216" alt="Screenshot 2024-02-07 at 17 31 32" src="https://github.com/vercel/community/assets/156811434/1a28a97a-ba88-4590-871b-451db7d8aca0">

**Comment by amyegan** [comment, marked as answer]
Hey @hewb-dev-team. A similar issue was reported in https://github.com/vercel/community/discussions/5449. Please upgrade to Next.js version 14.1 and let me know if you still have trouble with it 🙏

**Comment by leerob** [comment]
For posterity, I ran into a similar issue here where I was importing a component with the name `Table`, and then the default export of my component was also named `Table`. Might help!

**Comment by hewb-dev-team** [comment]
I upgraded and it s fixed thank you
